dictionary.py

```diff
@@ -92,27 +85,14 @@
 
 # 6 Посчитать, сколько денег в месяц уходит на каждый отдел – и вывести вместе с названием отдела 
 
-HR_dep_spendings = list()
-IT_dep_spendings = list()
+
 for dictionary in departments:
-    
+    dep_spendings = list()
     for dicts in dictionary["employers"]:
         sallary = dicts["salary_rub"]
-        if dictionary['title'] == "HR department":
-            HR_dep_spendings.append(sallary)  
-        else:
-            for tax in taxes:
-                if tax["department"] == "IT Department":
-                    tax_ = tax["value_percents"]
-                    IT_dep_spendings.append(sallary)
+        final_dep_spendings = dep_spendings.append(sallary)
```

Comment:
тут в final_dep_spendings присваивается None, потому что .append() ничего не возвращает
```suggestion
        dep_spendings.append(sallary)
```